media/js/firefox/new/experiment-firefox-new-batm-anim.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment_firefox_new_batm_anim',
+        variations: {
+            'xv=batmprivate': 33,   // Control: basic page, no machine
```

Comment:
This still won't work. Hitting this page never triggers the traffic cop experiment, since it's listed as a variation (at least, it never works for me locally). You can't have `batmprivate` listed here if it's the page that acts as the traffic cop.